CHANGELOG.asciidoc

```diff
@@ -3,6 +3,442 @@
+
+*Journalbeat*
+
+- Improve parsing of syslog.pid in journalbeat to strip the username when present {pull}16116[16116]
+
+*Metricbeat*
+
+- Add new dashboard for VSphere host cluster and virtual machine {pull}14135[14135]
+
+*Packetbeat*
+
+- Added redact_headers configuration option, to allow HTTP request headers to be redacted whilst keeping the header field included in the beat. {pull}15353[15353]
+- Add dns.question.subdomain and dns.question.top_level_domain fields. {pull}14578[14578]
+
+==== Bugfixes
+
+*Affecting all Beats*
+
+- Fix Kubernetes autodiscovery provider to correctly handle pod states and avoid missing event data {pull}17223[17223]
+- Fix `add_cloud_metadata` to better support modifying sub-fields with other processors. {pull}13808[13808]
+- Fix panic in the Logstash output when trying to send events to closed connection. {pull}15568[15568]
+- Fix logging target settings being ignored when Beats are started via systemd or docker. {issue}12024[12024] {pull}15422[15442]
+- Fix issue where default go logger is not discarded when either * or stdout is selected. {issue}10251[10251] {pull}15708[15708]
+- Remove superfluous use of number_of_routing_shards setting from the default template. {pull}16038[16038]
+- Fix index names for indexing not always guaranteed to be lower case. {pull}16081[16081]
+- Upgrade go-ucfg to latest v0.8.1. {pull}15937{15937}
+- Fix loading processors from annotation hints. {pull}16348[16348]
+- Upgrade go-ucfg to latest v0.8.3. {pull}16450{16450}
+- Add `ssl.ca_sha256` option to the supported TLS option, this allow to check that a specific certificate is used as part of the verified chain. {issue}15717[15717]
+- Fix `NewContainerMetadataEnricher` to use default config for kubernetes module. {pull}16857[16857]
+- Improve some logging messages for add_kubernetes_metadata processor {pull}16866{16866}
+- Fail to start if httpprof is used and it cannot be initialized. {pull}17028[17028]
+- Fix concurrency issues in convert processor when used in the global context. {pull}17032[17032]
+
+*Filebeat*
+
+- Fix mapping error when zeek weird logs do not contain IP addresses. {pull}15906[15906]
+- Fix merging of fileset inputs to replace paths and append processors. {pull}16450{16450}
+- Fix Elasticsearch `_id` field set by S3 and Google Pub/Sub inputs. {pull}17026[17026]
+- Fixed various Cisco FTD parsing issues. {issue}16863[16863] {pull}16889[16889]
+- Fix default index pattern in IBM MQ filebeat dashboard. {pull}17146[17146]
+
+*Metricbeat*
+
+- Change lookup_fields from metricset.host to service.address {pull}15883[15883]
+- Fix skipping protocol scheme by light modules. {pull}16205[pull]
+- Made `logstash-xpack` module once again have parity with internally-collected Logstash monitoring data. {pull}16198[16198]
+- Fix detection and logging of some error cases with light modules. {pull}14706[14706]
```

Comment:
Not sure this will make sense to end users because AFAIK we don't describe light modules anywhere. Also seems vague to say "some error cases".